UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -36,7 +36,7 @@ class UniConnectUnitTest {
     @get:Rule
     var rule: TestRule = InstantTaskExecutorRule()
     lateinit var universityService: IUniversityService
-    var allUniversities : List<University>? = ArrayList<University>()
+    var allUniversities: List<University>? = ArrayList<University>()
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide